Ja-1103-Word-game_starter.py

```diff
@@ -0,0 +1,43 @@
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list, what letters have been guessed so far
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    # FILL IN YOUR CODE HERE...
+    """
+    for every letter in secret word
+      if i is not in the letters guessed
+        return false
+    return true
+    """
+    for i in secret_word:
+      if i not in letters_guessed:
+        return False
+    return True
+
+
+### Testcases
+print(is_word_guessed('apple', ['a', 'e', 'i', 'k', 'p', 'r', 's']))
+# print(is_word_guessed('durian', ['h', 'a', 'c', 'd', 'i', 'm', 'n', 'r', 't', 'u']))
+# print(is_word_guessed('pineapple', []))
+
+
+
+def get_guessed_word(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list, what letters have been guessed so far
+    returns: string, comprised of letters and underscores that represents
+      what letters in secret_word have been guessed so far.
+    '''
+    # FILL IN YOUR CODE HERE...
+    full_word = ''
+    for i in secret_word:
+      if i in letters_guessed:
+        full_word += i
+        pass
+      else: 
+        fullword += ""

```

Comment:
When inserting underscores into your string, it's a good idea to add at least a space after each 
one, so it's clear to the user how many unguessed letters are left in the string (compare the 
readability of 
____
with 
_ _ _ _ 
).